src/Http/Requests/ResourceQuery.php

```diff
@@ -104,9 +105,9 @@ public static function queryOne(string $resourceType): QueryParameters
      * Perform resource authorization.
      *
      * @param Authorizer $authorizer
-     * @return bool
+     * @return bool|Response
      */
-    public function authorizeResource(Authorizer $authorizer): bool
+    public function authorizeResource(Authorizer $authorizer): bool|Response
```

Comment:
This change is technically breaking. As the method is `public`, anyone could be calling it - and if they're relying on it returing a boolean it'll break their implementation. 

So I'll need to tag this as a major release.